content/articles/setting-up-kali-linux-for-dual-boot/index.md

```diff
@@ -0,0 +1,124 @@
+
+In this tutorial, we will learn how to set up `Kali Linux` onto a machine already installed with Windows. This process allows you to safely keep your data initially on Windows.
+> **Note:** This process is similar to any other Linux-based operating system.
+ 
+### Prerequisites
+For a successful setup, you need the following:
+1. An 8GB or higher pen drive.
+2. Minimum space of 25GB on your Hard drive.
+3. Windows OS running on your PC or laptop.
+
+### Getting started
+Now, download the files required during the process.
+
+#### Downloading Kali Linux Iso image
+Navigate to the [Kali Linux official site](https://kali.org/get-kali/#kali-bare-metal) and download the latest version of Kali Linux.
+
+#### Creating a bootable pen drive
+We will first download software used to make a pen drive bootable. We'll use `Rufus` in this context. You can download [Rufus here](http://rufus.ie/en_US/).
+
+Open Rufus and insert the pen drive onto your machine. 
+
+Select the appropriate options depending on the pen drive and the OS you want to install.
+
+Ensure that the pen drive is empty as the process will format the drive. Carefully select the options appropriately to avoid any errors at any point.
+
+![Pen Drive Options](/engineering-education/setting-up-kali-linux-for-dual-boot/rufus.png)
+
+Patiently wait as the process might take a while depending on your pen drive.
+
+### Creating space for Kali Linux
+On your Windows machine, navigate to the disk management options by searching `computer management` on the search bar. Select the primary disk where Windows OS is located.
+
+Right-click and allocate the amount of space suitable for Kali Linux. The partition should appear as shown below:
+
+![Shrinking Primary Disk](/engineering-education/setting-up-kali-linux-for-dual-boot/shrink.png)
+
+### Installing Kali Linux
+After successfully shrinking the space, install Kali Linux on the machine as follows:
+
+Reboot your machine and access the boot menu. You can access the boot menu by either clicking the `esc`, `f2`, `f10`, or `f12` keys depending on the architecture of your machine. Follow the directives from the manufacturer for more information.
+
+Select the pen drive listed on the menu to boot your PC from it. 
+
+Choose the `options` as guided below to avoid any errors during the installation process.
+
+- Select **Graphical Installation**.
+- Choose **Language**- in my case, `English`.
+- Select your **Region** and your preferred **Keyboard lab**.
+
+Wait for the installation set-up to load all components. This should take a short while.
+
+In the following setup, we'll use the recommended choices for an assured successful set-up.
+
+- Load missing firmware from removable media? - `No`.
+- Connect to your WiFi card if one is available - This is optional.
+
+Wait for the set-up to `configure all the Networks`. This process is also quick.
+
```

Comment:
This only happens when the user decides to connect to a WI-FI network. Please clarify that.